api/rooms/views/main.py

```diff
@@ -0,0 +1,22 @@
+import requests, json
+
+from django.views     import View
+from django.http      import JsonResponse
+
+from my_settings import API_KEY
+
+class SearchRoomView(View):
+    def get(self, request):
+        try:
+            search_address = request.GET.get('search_address', None)
```

Comment:
`address`만으로 충분한 것 같습니다. 그리고 `.get()` 호출 시에 default로 None을 반환하기 때문에 적어주지 않아도 됩니다~

```suggestion
            address = request.GET.get('address')
```

Reply:
넵!